system/ORM/Relation/BelongsTo.php

```diff
@@ -20,15 +20,22 @@ class BelongsTo extends Relation
     protected $primaryKey;
 
 
-    public function __construct($className, $primaryKey)
+    public function __construct($$className, Model $model, $otherKey = null)
```

Comment:
Parse error: syntax error, unexpected '$', expecting variable (T_VARIABLE) in /home/travis/build/simple-mvc-framework/framework/system/ORM/Relation/BelongsTo.php on line 23